app/views/home/index.html.slim

```diff
@@ -1,2 +1,37 @@
-h1 Home#index
-p Find me in app/views/home/index.html.slim
+.container
+  h1 Investment tracking
+
+.entries
+  .container-fluid
+    .row
+      .col-md-2
+      .col-md-6
+        p Your open investment time in hours:
+      .col-md-2
+        p Your rate to total worked hours:
+      .col-md-2
+        p Your used investment time:
+    .row
+      hr
+    - @sorted_entries.each do |entries|
+      .row
+        .col-md-2
+          p = entries['name'].force_encoding('ISO-8859-1').encode('UTF-8')
+        .col-md-6
+          .progress
+            - if entries['open_investment_time'] >= 60
+              .progress-bar.progress-bar-danger[role='progressbar'
+              style='width: #{entries['calculation_progress_bar']}%']
+                = entries['open_investment_time'].to_s + 'h'
+            - if entries['open_investment_time'] >= 40 && entries['open_investment_time'] < 60
+              .progress-bar.progress-bar-warning[role='progressbar'
+              style='width: #{entries['calculation_progress_bar']}%']
+                = entries['open_investment_time'].to_s + 'h'
+            - if entries['open_investment_time'] < 40
+              .progress-bar.progress-bar-success[role='progressbar'
+              style='width: #{entries['calculation_progress_bar']}%']
+                = entries['open_investment_time'].to_s + 'h'
```

Comment:
This could be [dryer](https://en.wikipedia.org/wiki/Don't_repeat_yourself). I'd expect only the indication class to change depending from the number of hours. Some more input here: https://mixandgo.com/blog/the-beginners-guide-to-rails-helpers